core/src/main/java/nl/nn/adapterframework/scheduler/JobDef.java

```diff
@@ -784,44 +784,50 @@ private void checkReload(IbisManager ibisManager) {
 										+ configVersion
 										+ "] will be reloaded with new version ["
 										+ ibisConfigVersion + "]");
-								configsToReload.add(configName);
+								configNamesToReload.add(configName);
 							}
 						}
 					}
 				}
 			} catch (Exception e) {
-				getMessageKeeper().add("error while executing query [" + selectQuery	+ "] (as part of scheduled job execution)", e);
+				errorMsg = "error while executing query [" + selectQuery + "] (as part of scheduled job execution)";
+				getMessageKeeper().add(errorMsg, e);
 			} finally {
 				qs.close();
 				JdbcUtil.fullClose(conn, rs);
 			}
 
-			if (!configsToReload.isEmpty()) {
-				for (String configToReload : configsToReload) {
-					ibisManager.getIbisContext().reload(configToReload);
-				}
-			}
-			
-			if (CONFIG_AUTO_DB_CLASSLOADER) {
-				// load new (activated) configs
-				List<String> dbConfigNames = null;
-				try {
-					dbConfigNames = ConfigurationUtils.retrieveConfigNamesFromDatabase(ibisManager.getIbisContext(), configJmsRealm, true);
-				} catch (ConfigurationException e) {
-					getMessageKeeper().add("error while retrieving configuration names from database", e);
-				}
-				if (dbConfigNames != null && !dbConfigNames.isEmpty()) {
-					for (String currentDbConfigurationName : dbConfigNames) {
-						if (!configNames.contains(currentDbConfigurationName)) {
-							ibisManager.getIbisContext().load(currentDbConfigurationName);
-						}
+			if (errorMsg == null) {
```

Comment:
Ik kan niet goed beoordelen waarom je dit zo hebt aangepast, zou jij dit toe kunnen lichten?

Reply:
Bij de IJA_API kwam dezelfde configuratie soms meerdere malen voor. Dit kwam omdat tijdens het ophalen van alle configuraties, die reload moeten worden, een fout optrad en hierdoor de tabel met alle te reloaden configuraties niet compleet/correct was. Om dit te voorkomen wordt een reload enkel gedaan als we zeker weten dat de tabel met alle te reloaden configuratie klopt.